README.md

```diff
@@ -1,10 +1,53 @@
 Bowling Challenge in Ruby
 =================
+## The Task
+Count and sum the scores of a bowling game for one player.
+
+## How to use:
+-------
+* Clone this repo.
+* Run the command 'bundle' in the project directory to ensure you have all the gems.
+* Run tests with RSpec.
+* Run tests with Rubocop.
+* Run with irb in terminal, `require './lib/frame'` to test some frames by inputting the scores. For example `frame.first_roll(4)` ...
+
+## Process
+
+![domain model](images/domain_model_03.png)
+=================
+
+![class model](images/class_diagram_01.png)
+=================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+### Frame class - IRB tests:
+
+Strike
+- 001 > require './lib/frame'
+- => true 
+- :002 > frame = Frame.new
+- => #<Frame:0x00007f874d0bf820> 
+- :003 > frame.first_roll(4)
+- => "Player to roll again" 
+- :004 > frame.first_roll(10)
+- => "STRIKE! End of frame" 
+
+Spare
+- 3.0.0 :002 > frame = Frame.new
+-  => #<Frame:0x00007fd6dfa121c0 @pinfall={:roll1=>nil, :roll2=>nil}, @scorecard=[], @bonus=nil> 
+- 3.0.0 :003 > frame.first_roll(4)
+-  => "Player to roll again" 
+- 3.0.0 :004 > frame.second_roll(6)
+-  => "SPARE!" 
+- 3.0.0 :005 > frame.first_roll(4)
+-  => "Player to roll again" 
+- 3.0.0 :006 > frame.second_roll(6)
+-  => "SPARE!" 
+- 3.0.0 :007 > frame.scorecard
+-  => [{:roll1=>4, :roll2=>6, :bonus_points=>4}, {:roll1=>4, :roll2=>6}] 
+- 3.0.0 :008 > 
+
+Work in progress
+- I have written a test to start looking at playing a full game of bowling and printing a scoresheet. I have started writing a test, but I need to think more about how I want to display the info. When I have more time I will pick it up from here.
```

Comment:
You clearly stated where you go to which is great, also having irb lines was useful to see how the problem was approached.